test/wrapper_test.rb

```diff
@@ -2879,14 +2879,14 @@ def test_number_of_service_vrps_generated_in_split_independent
 
   def test_split_independent_vrps_with_useless_vehicle
     vrp = TestHelper.create(VRP.independent_skills)
-    vrp.vehicles << Models::Vehicle.create(id: 'useless_vehicle')
+    vrp.vehicles << Models::Vehicle.create(id: 'useless_vehicle', matrix_id: 'matrix_0')
```

Comment:
How did this and the next test pass without this correction O_O 

Without the correction the new vehicle has no matrix, so it should lead to a matrix/router call but tests were passing. I am confused, any idea?

Reply:
This vehicle was put in a sub problem without services :/ So it never tried to load the matrix. But now, as there is a matrix is the problem and we build the sub_problems with an update of the matrix links, it fails.